routes/global_state.go

```diff
@@ -345,6 +345,11 @@ type UserMetadata struct {
 
 	// Txn hash in which the referrer was paid
 	ReferrerDeSoTxnHash string
+
+	// The number of unread notifications stored in the db.
+	UnreadNotifications uint64
+	// The most recently scanned notification transaction index in the database. Stored in order to prevent unnecessary re-scanning.
+	LatestUnreadNotificationIndex int64
```

Comment:
There is an index in this struct called, "NotificationLastSeenIndex".  What's the difference between these two variables?  If it's necessary to have both, we should probably clarify or mark one as deprecated (unless it is used in the bitclout.com frontend).

Reply:
`NotificationLastSeenIndex` tracks the last notification that a user has read, and allows the backend to determine which notifications are therefore "unread"

`LatestUnreadNotificationIndex` tracks the last index that the backend has scanned via the `getUnreadNotifications` route. By tracking this index, we can ensure that we don't duplicate work, and that we only scan new notifications once for this route.